lxqt-admin-time/timedatectl.cpp

```diff
@@ -0,0 +1,44 @@
+#include "timedatectl.h"
+#include <QProcess>
+#include <QDebug>
+
+TimeDateCtl::TimeDateCtl()
+{
+    QProcess timedatectl;
+    timedatectl.start(QStringLiteral("timedatectl"));
+    timedatectl.waitForFinished();
+    while(!timedatectl.atEnd()) {
+        QString line = timedatectl.readLine().trimmed();
+        int findTZ = line.indexOf(QLatin1String("Time zone:"));
+        if(findTZ != -1) {
+            findTZ += 10;
+            while(line[findTZ] == ' ')
+                ++findTZ;
+            int space = line.indexOf(' ', findTZ);
+            mTimeZone = line.mid(findTZ, space - findTZ);
+            break;
+        }
+    }
+    timedatectl.close();
+}
+
+bool TimeDateCtl::commit()
```

Comment:
Here I think we should report the problem (stdout/stderr) in case of failure. Now there is no way for end user to know why the tool does nothing (in case of failure).